unit-tests/py/rspy/usb_relay.py

```diff
@@ -0,0 +1,268 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+import re
+from rspy import log
+
+NO_RELAY = 0
+ACRONAME = 1
+YKUSH = 2
+
+
+class usb_relay:
```

Comment:
Please add docs

Reply:
Also, the name:
Why USB?

Reply:
I'm emphasizing this: I don't think the word `relay` has any meaning, and I don't think the type (`usb`) has any justification: right now it's usb, but it could also be mipi, for example, or ethernet.

We're talking to a hub.